### PR TITLE
Making the intl-tel-input accessible

### PR DESCRIPTION
**Pull request by isramv**
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

And base on the work of: 

~https://github.com/jackocnr/intl-tel-input/pull/1002/files~ this PR was merged recently, `master` got upgraded on this repo.

### CODE REVIEW

File: src/js/intlTelInput.js
Comment (agarzola) [inline src/js/intlTelInput.js:513]: `aria-expanded` should only be applied to the element that toggles the expandable/collapsible content, not to the content itself. [Here are some examples from WCAG Working Group](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region) showing how to use `aria-expanded` correctly. It looks like this library already sets `aria-expanded` on the selected flag, which is the toggle, so we do not need this line:
```suggestion
```

File: src/js/intlTelInput.js
Comment (agarzola) [inline src/js/intlTelInput.js]: Looks like this value is already set a couple of lines below, so we shouldn’t need this change. I suggest removing it.

**Review by agarzola — CHANGES_REQUESTED** [review]
See comments below.

File: src/js/intlTelInput.js
Comment (agarzola) [reply src/js/intlTelInput.js:513]: I just realized my mistake above. You’re _not_ setting `aria-expanded="true"` on the `countryList` element, you’re setting it on the `selectedFlag` element!

Comment (agarzola) [reply src/js/intlTelInput.js:513]: Resolving this conversation, but leaving the other one pending.

File: src/js/intlTelInput.js
Comment (isramv) [reply src/js/intlTelInput.js]: @agarzola thanks I just removed the offending line.

**Comment by isramv** [comment]
@agarzola shall we create a tag and a release after the merge, and point to that release on MJ?

**Comment by agarzola** [comment]
> @agarzola shall we create a tag and a release after the merge, and point to that release on MJ?

@isramv I don’t think that’s necessary here. I don’t expect we’ll be making so many updates to this library that we’ll need to keep track of new versions. I also think there is a valuable contribution in the `aria-haspopup` change in this PR, which is not present in the original library.

I’d like to get this merged to unblock our own work, but once the dust has settled on this our own MJ work related to this, we should discuss an updated contribution to `jackocnr/intl-tel-input`.

Reviewing this now.

**Review by agarzola — CHANGES_REQUESTED** [review]
@isramv The change you made looks good to go, but I think you need to re-generate the `build/js/*` files to complete this PR.

**Review by gidhon — APPROVED** [review]

**Pull request merged**
